Jasmine/Common/Constants.swift

```diff
@@ -78,6 +78,14 @@ enum Constants {
 
             static let tileFallInterval: TimeInterval = 0.5
         }
+
+        enum GameOver {
```

Comment:
No, I think this is UI facing. Unless there is utility in having highly customised text for each game.